pkg/cnb/create_builder_test.go

```diff
@@ -168,12 +167,9 @@ func testCreateBuilderOs(os string, t *testing.T, when spec.G, it spec.S) {
 			},
 		}
 
-		subject = RemoteBuilderCreator{
-			RegistryClient:         registryClient,
-			LifecycleImage:         lifecycleImage,
-			KpackVersion:           "v1.2.3 (git sha: abcdefg123456)",
-			NewBuildpackRepository: newBuildpackRepo,
-		}
+		lifecycleProvider *fakeLifecycleProvider
+
+		subject RemoteBuilderCreator
```

Comment:
Can we still initialize the remote builder creator here?

Reply:
updated